src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -760,6 +770,12 @@ def __init__(self, config):
         # Initialize weights and apply final processing
         self.post_init()
 
+    def tie_weights(self):
+        super(BloomForCausalLM, self).tie_weights()
```

Comment:
```suggestion
        super().tie_weights()
```
Let's write Python3 code pretty please.